src/mobile-app/components/experiment-picker-item.tsx

```diff
@@ -10,10 +10,20 @@ interface IProps {
 
 export const ExperimentPickerItem: React.FC<IProps> = ({experiment, setExperiment}) => {
   const {name} = experiment.metadata;
+  const buttonColor = (initials: string) => {
```

Comment:
I think this is ok for now but we should add this to the schema in the future instead of hardcoding it in the css.